manticore/seth.py

```diff
@@ -54,13 +54,13 @@ class IntegerOverflow(Detector):
         Detects any it overflow on instructions ADD and SUB.
     '''
     def did_evm_execute_instruction_callback(self, state, instruction, arguments, result):
-        if instruction.semantics == 'ADD':
+        mnemonic = instruction.semantics
+        if mnemonic in ('ADD', 'MUL'):
```

Comment:
Should this support `ADDMOD` and `MULMOD` as well?

Reply:
instruction.is_arithmetic == True may imply we could check for something.
div_by_zero is probably a finding too. There are a bunch of low hanging fruit there.

Reply:
i think it's okay to not include addmod/mulmod if we note it somewhere. adding those checks is a little less straightforward.

Reply:
i added https://github.com/trailofbits/manticore/issues/657 to track it